src/main/java/com/autotune/analyzer/experiment/ExperimentResultValidation.java

```diff
@@ -79,7 +90,10 @@ public void validate(List<ExperimentResultData> experimentResultDataList, Map<St
                         if (null != kruizeObject.getResultData())
                             isExist = kruizeObject.getResultData().contains(resultData);
                         if (isExist) {
-                            errorMsg = errorMsg.concat(String.format("Experiment name : %s already contains result for timestamp : %s", resultData.getExperiment_name(), resultData.getEndtimestamp()));
+                            DateFormat dateFormat = new SimpleDateFormat(KruizeConstants.DateFormats.STANDARD_JSON_DATE_FORMAT);
+                            dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
Are we assuming that the input date is always in UTC timezone?

Reply:
Updated now.